test/get.sh

```diff
@@ -410,18 +410,18 @@ it_honors_the_parameter_flags_for_submodules() {
   "file://"$project_folder 1 "all" true $dest_recursive_true |  jq -e "
     .version == {ref: $(echo $project_last_commit_id | jq -R .)}
   "
-  test "$(git -C $project_folder rev-parse HEAD)" = $project_last_commit_id
-  test "$(git -C $dest_recursive_true/$submodule_name rev-list --all --count)" = 1
-  test "$(git -C $dest_recursive_true/$submodule_name/$subsubmodule_name rev-list --all --count)" = 1
+  test "$(git -C $dest_recursive_true rev-parse HEAD)" = $project_last_commit_id
+  test "$(ls $dest_recursive_true/$submodule_name | wc -l)" = 2
+  test "$(ls $dest_recursive_true/$submodule_name/$subsubmodule_name | wc -l)" = 1
```

Comment:
What're these changes for? Just curious.

Reply:
Hi, there are some details in the commit message for a95a98a, but essentially, this updates these to test the behaviour that differs around the submodule_recursive flag, which is the number of files present as opposed to the commit depth. It looks as though the original assertions may have been copied from the tests for the depth flag.

Reply:
Ah, thanks for the descriptive commits!